### ISSUE TITLE
Windows support

### ISSUE THREAD

**Issue by Marv51** [s3-actions/s3cmd#5]
I tried to run the s3cmd action on Windows in the GitHub Action `windows-2022` environment.

This seems unsupported right now. 

It would be cool to add support, or if that is not possible, maybe the error message could be improved?

Current error message:
```
The system cannot find the path specified.
child_process.js:674
    throw err;
    ^

Error: Command failed: /bin/bash -c 'pip3 install s3cmd --no-cache'
The system cannot find the path specified.
```

**Comment by bluebrown** [comment]
Good point. I had this always in the back of my head and wanted to add it to the readme, but never did for some reason.

The reason why windows is not supported is that we can run it without docker on Linux machines. Perhaps it could be done on Windows as well, but I am not sure if python is pre-installed there.

If you want to experiment with installing s3cmd on a windows machine based on the current [pattflorm](https://nodejs.org/api/process.html#process_process_platform), it would make a nice pull request.

Otherwise, we could think about using docker but I kind of dislike the idea because it makes the build more slow.

**Comment by Marv51** [comment]
I can't get s3cmd to properly work in my Github Action on Windows. Locally it is quite straight forward, but for whatever reason it just does not work on GitHub Actions. (I have moved on to the official AWS SDK which comes preinstalled)

This is what I found:

plain `pip install s3cmd` seems to work on Windows.
Also I believe the settings file should be in "$env:APPDATA\s3cmd.ini".

```yaml
- name: Install S3cmd
        run: |
          pip install s3cmd
          New-Item "$env:APPDATA\s3cmd.ini"
          Set-Content "$env:APPDATA\s3cmd.ini" '[default]'
          Add-Content "$env:APPDATA\s3cmd.ini" "access_key = ${{ secrets.DEPLOY_ACCESSKEY }}"
          [...]
```

However, the pip packages are not properly added to the PATH by default. And even if you add it to the path, powershell can't execute the "s3cmd" file with python automatically.

So instead I used: 
```
$s3location = $(python -c 'import os,sysconfig;print(sysconfig.get_path(\"scripts\",f\"{os.name}_user\"))')
$s3cmd = $s3location + "/s3cmd"
python $s3cmd ls
```

for whatever reason, with that my github action **just keeps running** with that and **never terminates**.
There is no output, or debug output. Locally that also works fine.

Hey, I know that is not a great contribution "I tried a bunch of things and I couldn't get it to work" but maybe this helps someone skip a bunch of steps and start straight where I gave up.
